addons/website_event_exhibitor/controllers/exhibitor.py

```diff
@@ -110,6 +110,7 @@ def _event_exhibitors_get_values(self, event, **searches):
             'hide_sponsors': True,
             # search information
             'searches': searches,
+            'search_count': len(sponsors),
```

Comment:
Above, sponsors is redefined in the loop:
```python
for sponsor_category, sponsors in sponsor_categories_dict.items():
```
So if there are results in different categories, the count will only count the ones in the last category.
See https://www.awesomescreenshot.com/image/45177702?key=7a511f07067c083eff765ffb0950940b

So we should probably rename the first `sponsors` to `sorted_sponsors` for instance, as we sort it in its definition. And then do not forget to update the one here: `for sponsor in sponsors:` and finally the one in the len(...) you introduced.

Reply:
Hello @Odoonan 

Changes are done. 
You can check it if you want.
Just waiting for the Runbot :slightly_smiling_face: 

Thanks